zulipterminal/ui_tools/views.py

```diff
@@ -1749,3 +1749,53 @@ def keypress(self, size: urwid_Size, key: str) -> str:
             )
             return key
         return super().keypress(size, key)
+
+
+class FullRawMsgView(PopUpView):
+    def __init__(
+        self,
+        controller: Any,
+        message: Message,
+        topic_links: "OrderedDict[str, Tuple[str, int, bool]]",
+        message_links: "OrderedDict[str, Tuple[str, int, bool]]",
+        time_mentions: List[Tuple[str, str]],
+        title: str,
+    ) -> None:
+        self.controller = controller
+        self.message = message
+        self.topic_links = topic_links
+        self.message_links = message_links
+        self.time_mentions = time_mentions
+        max_cols, max_rows = controller.maximum_popup_dimensions()
+
+        # Get rendered message header and footer
+        msg_box = MessageBox(message, controller.model, None)
+
+        # Get raw message content widget list
+        response = controller.model.fetch_raw_message_content(message["id"])
+        if response is None:
+            controller.report_error("Error: Couldn't fetch raw message content")
```

Comment:
This is going to be a duplicate of the error handling in the model method?

Reply:
Ah yes. I'll simply return in that case :+1:. Thanks for pointing out!